fix: Fix inventory and update issues

pacampbell opened the pull request:
- Fixed an issue where items are unable to be equipped after
  transitioning between BBM and Lestania.
- Fixed issue where pawn items get equipped to the player during limit
  break.
- Added emblem stats to S2CCharacterSwitchGameModeNtc
- Fixed packet name for logging
- Renamed some unk fields to be EmblemStatList
- Updated item safety notice

Note: Stats raised with emblem points not currently working if the emblem is equipped to the pawn.

# Checklist:
- [x] The project compiles
- [x] The PR targets `develop` branch

pacampbell committed:
fix: Fix inventory issue

- Fixed an issue where items are unable to be equipped after
  transitioning between BBM and Lestania.
- Fixed issue where pawn items get equipped to the player during limit
  break.
- Added emblem stats to S2CCharacterSwitchGameModeNtc
- Fixed packet name for logging
- Renamed some unk fields to be EmblemStatList

RyanYappert commented on Arrowgene.Ddon.Shared/Model/ItemNoticeType.cs:
```diff
@@ -55,29 +55,33 @@ public enum ItemNoticeType : ushort
-        // 0x44
-        // 0x45
+        // 0x44 // Item consumed
+        ItemSafetySetting = 0x45,
```
Can you adjust the notice used in `ItemChangeAttrDiscardHandler` to use this update type? It calls `ItemManager::SetSafetySetting`.

alborrajo commented on Arrowgene.Ddon.GameServer/Handler/CharacterSwitchGameModeHandler.cs:
```diff
@@ -82,68 +74,35 @@ public override S2CCharacterSwitchGameModeRes Handle(GameClient client, C2SChara
+                UpdateType = ItemNoticeType.ChangeEquip,
+                UpdateItemList = SwapCharacterInventories(client.Character, previousStorage, [StorageType.CharacterEquipment])
+            });
+#endif
```
dead code? 🤔

pacampbell committed:
fix: Fix issue with skill EX ids

- Fixed issue with EX skill id releases being inverted.
- Added safety lock update notice.
- Added emblem stat list to a few more places.
